### PR TITLE
CFM-970: Adds "Enable notifications" link to the CTA block

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-970
Enable notifications (at the bottom of the pic):
enable-notifications
Disable notifications (at the bottom of the pic):
disable-notifications

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module [hunk 1]
```diff
@@ -253,6 +253,11 @@ function c4m_features_og_group_dashboard_block_info() {
     'cache' => DRUPAL_CACHE_PER_PAGE,
   );
 
+  $blocks['c4m_follow_block'] = array(
+    'info' => t('C4M Follow group block'),
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @DavidHernandez 
This issue talks about Projects and not groups

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: I'm wondering if we should have a `c4m_features_og_project_dashboard` as well
@jover @soniCaH @mfernea @janlemmens 
 ?

Comment (DavidHernandez) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @ordavidil I've updated the title to say "C4M Follow group or project block" as this block can be used for both content types.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: It is probably should follow the current context

Comment (DavidHernandez) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @ordavidil this PR handles both cases, adding the Follow link to Projects and to Groups (see the changes on this file: https://github.com/capacity4dev/capacity4more/blob/cccb39c7a6363533768a63817d8e0ebbd8a3164c/project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.pages_default.inc )

I had issues to handle this functionality in both Projects and Groups ( https://issuetracker.amplexor.com/jira/browse/CFM-1358  and https://issuetracker.amplexor.com/jira/browse/CFM-1360 ) and I was told that might be better to tackle them together. Both issues are referencing this PR.

Should I split the PR in two PRs?

**Comment by soniCaH** [comment]
Functionality working.

Please update styling to match design and add it under the existing links/CTA's (including icon + updated text):

screen shot 2016-08-22 at 17 37 04

File: project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module [hunk 2]
```diff
@@ -264,11 +264,9 @@ function c4m_features_og_group_dashboard_block_info() {
 function c4m_features_og_group_dashboard_block_view($delta = '') {
   $block = array();
 
-  if ($delta != 'c4m_cta_block') {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @DavidHernandez 
why do we need to change that ?

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @DavidHernandez ^^

Comment (DavidHernandez) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: Actually, we don't need that check as there is only one block left on the module.

Comment (DavidHernandez) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module]: @ordavidil that hook is only invoked when a block defined by that module is rendered. There is only one block on the module: c4m_cta_block. This function will be called with "$delta = 'c4m_cta_block'". So, that if is not necessary.

**Comment by jover** [comment]
Had to fix a typo which prevented the link being rendered in the block...

**Pull request merged**